apis/objectstorage.k8s.io/v1alpha1/types.go

```diff
@@ -21,63 +21,81 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-type BucketRequestSpec struct {
-	// +optional
-	BucketInstanceName string `json:"bucketInstanceName,omitempty"`
-	// +optional
-	BucketPrefix string `json:"bucketPrefix,omitempty"`
-	// +optional
-	BucketClassName string `json:"bucketClassName,omitempty"`
-}
+func init() {
+	SchemeBuilder.Register(&Bucket{}, &BucketList{})
+	SchemeBuilder.Register(&BucketRequest{}, &BucketRequestList{})
+	SchemeBuilder.Register(&BucketClass{}, &BucketClassList{})
 
-type BucketRequestStatus struct {
-	// +optional
-	Message string `json:"message,omitempty"`
-	// +optional
-	BucketAvailable bool `json:"bucketAvailable"`
+	SchemeBuilder.Register(&BucketAccess{}, &BucketAccessList{})
+	SchemeBuilder.Register(&BucketAccessRequest{}, &BucketAccessRequestList{})
+	SchemeBuilder.Register(&BucketAccessClass{}, &BucketAccessClassList{})
 }
 
-type AnonymousAccessMode struct {
-	// +optional
-	Private bool `json:"private,omitempty"`
-	// +optional
-	PublicReadOnly bool `json:"publicReadOnly,omitempty"`
+type RetentionPolicy string
```

Comment:
The lastest kep pr defines this field as "deletionPolicy" based on review meeting consensus.

Reply:
Good catch. I’ll update it